src/shared/sleep-config.h

```diff
@@ -40,7 +40,7 @@ DEFINE_TRIVIAL_CLEANUP_FUNC(SwapEntry*, swap_entry_free);
  * and the matching /proc/swap entry.
  */
 typedef struct HibernateLocation {
-        char *resume;
+        dev_t resume;
```

Comment:
btw, i think "resume" is a bit weird name for this. This refers to a device, no? The verb "resume" doesn't really transport that fact... maybe call it "devno"?

Reply:
I was just trying to match the naming of the `/sys` values; `resume` and `resume_offset`. I went ahead and changed them as you suggested.